src/components/BorrowerProfile/LoanProgress.vue

```diff
@@ -29,7 +29,7 @@
 						{{ progressPercent | numeral('0%') }} funded
 					</p>
 				</div>
-			</div>
+			</template>
 			<div v-else>
```

Comment:
I think this also needs to be changed from div to template for funded loans.

Reply:
Yeah, I tested that out but it shows everything in the same row while design has two rows for `This loan is fully funded!` and the other section